src/pages/Rockets.jsx

```diff
@@ -1,9 +1,55 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRockets, reserveRocketAction } from './RocketSlice';
+import './rockets.css';
 
-const Rockets = () => (
-  <>
-    <h1>Rocket page</h1>
-  </>
-);
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const status = useSelector((state) => state.rockets.status);
+  const error = useSelector((state) => state.rockets.error);
```

Comment:
you might want to use a deconstructor here to avoid repeating yourself as shown below:

 ` const { rockets, status, error } = useSelector((state) => state.rockets);`

Reply:
Done